resources/packs/gcp/gcp.lr

```diff
@@ -556,43 +559,234 @@ private gcp.storage.bucket @defaults("id") {
+  // The maintenance window
+  maintenanceWindow gcp.project.sqlService.instance.settings.maintenanceWindow
+  // The local user password validation policy
+  passwordValidationPolicy gcp.project.sqlService.instance.settings.passwordValidationPolicy
+  // The pricing plan 
+  pricingPlan string
+  // The type of replication
+  replicationType string
+  // The version of instance settings
+  settingsVersion int
+  // SQL server specific audit configuration
+  sqlServerAuditConfig dict
+  // Configuration to increase storage size automatically
+  storageAutoResize bool
+  // The maximum size to which storage capacity can be automatically increased
+  storageAutoResizeLimit int
+  // The tier of service for this instance
+  tier string
+  // Server timezone
+  timeZone string
+  // User-provided labels
+  userLabels map[string]string
+}
+
+// GCP Cloud SQL Instance Settings Backup Configuration
+private gcp.project.sqlService.instance.settings.backupconfiguration {
+  // Internal ID
+  id string
+  // Backup retention settings
+  backupRetentionSettings dict
+  // Whether binary log is enabled
+  binaryLogEnabled bool
+  // Whether this configuration is enabled
+  enabled bool
+  // The location of the backup
+  location string
+  // Whether point in time recovery is enabled
+  pointInTimeRecoveryEnabled bool
+  // Start time for the daily backup configuration in UTC timezone in the 24 hour format
+  startTime string
+  // The number of days of transaction logs we retain for point in time restore
+  transactionLogRetentionDays int
+}
+
+// GCP Cloud SQL Instance Settings Deny Maintenance Period
+private gcp.project.sqlService.instance.settings.denyMaintenancePeriod @defaults("startDate endDate") {
+  // Internal ID
+  id string
+  // Deny maintenance period end date.
+  endDate string
+  // Deny maintenance period start date.
+  startDate string
+  // Time in UTC when the deny maintenance period starts and ends
+  time string
+}
+
+// GCP Cloud SQL Instance Settings IP Configuration
+private gcp.project.sqlService.instance.settings.ipConfiguration {
+  // Internal ID
+  id string
+  // The name of the allocated IP range for the private IP Cloud SQL instance
+  allocatedIpRange string
+  // The list of external networks that are allowed to connect to the instance using the IP
+  authorizedNetworks []dict
+  // Whether the instance is assigned a public IP address
+  ipv4Enabled bool
+  // The resource link for the VPC network from which the Cloud SQL instance is accessible for private IP
+  privateNetwork string
+  // Whether SSL connections over IP are enforced or not
+  requireSsl bool
+}
+
+// GCP Cloud SQL Instance Settings Maintenance Window
+private gcp.project.sqlService.instance.settings.maintenanceWindow @defaults("day hour") {
+  // Internal ID
+  id string
+  // Day of week (1-7), starting on Monday
+  day int
+  // Hour of day - 0 to 23
+  hour int
+  // Maintenance timing setting: canary (Earlier) or stable (Later)
```

Comment:
Are "Earlier" and "Later" the values, and they're capitalized? Is that why you capitalized them here?

Reply:
I think it's related to this field in their UI 
![image](https://user-images.githubusercontent.com/16187050/211275476-c2b05349-011b-4cc5-a941-bcb36598d013.png)